pre-commit/pre-commit-scripts/post-checkout.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
```

Comment:
I think this should run on all files, staged and unstaged contrary to the other two. But not a huge issue as we are going to be moving away from post-checkout hooks https://github.com/MarkBind/markbind/issues/2020.

If there's an easy way, feel free to add it in though.

Reply:
Hmm I dun think pre-commit supports this 😢

Reply:
🙆‍♂️